### PR TITLE
{data}[GCC/11.3.0] flook v0.8.1

### PR DESCRIPTION
**Pull request by ahkole**
requires:
- [ ] https://github.com/easybuilders/easybuild-easyblocks/pull/3034

(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ahkole** [comment]
Note: the patch file is needed to fix the pkgconf template file `flook.pc.in` which doesn't correctly link to an external Lua library.

File: easybuild/easyconfigs/f/flook/flook-0.8.1-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+parallel = 1
+
+local_comp_flags = 'VENDOR="gnu" FFLAGS="$FFLAGS" CFLAGS="$CFLAGS"'
+buildopts = 'liball %s' % local_comp_flags
+
+sanity_check_paths = {
+    'files': ['include/flook.mod', 'lib/libflook.a', 'lib/libflookall.a'],
+    'dirs': [],
+}
```
Comment (ocaisa) [inline easybuild/easyconfigs/f/flook/flook-0.8.1-GCC-11.3.0.eb:31]: Given that you've gone to the effort to create the easyblock to include the pc files, you could move the creation of the build options in there, and also construct the sanity check there (and include the missing check that ensure the pc file is actually installed). You can also do the restriction of parallelism there. That would make this easyconfig totally minimal going forward.

Comment (ahkole) [reply easybuild/easyconfigs/f/flook/flook-0.8.1-GCC-11.3.0.eb]: Thanks for the suggestion. I have updated the easyblock and easyconfig

**Comment by ocaisa** [comment]
@boegelbot: please test @ generoso EB_ARGS="--include-easyblocks-from-pr=3034"

**Comment by ocaisa** [comment]
@boegelbot: please test @ jsc-zen2 EB_ARGS="--include-easyblocks-from-pr=3034"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen2 EB_ARGS="--include-easyblocks-from-pr=3034"

**Review by ocaisa — APPROVED** [review]
LGTM

File: easybuild/easyconfigs/f/flook/flook-0.8.1_flook.pc.in.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+diff --git a/flook.pc.in b/flook.pc.in
```
Comment (boegel) [inline easybuild/easyconfigs/f/flook/flook-0.8.1_flook.pc.in.patch:1]: @ahkole Patch description is missing, do you mind opening a follow-up PR that add a short description of why this patch is needed on top, and mentions yourself as author (or includes a reference to source of the patch)?

Note that this implies updating the checksum of the patch in the `flook` easyconfig as well.

Comment (ahkole) [reply easybuild/easyconfigs/f/flook/flook-0.8.1_flook.pc.in.patch:1]: @boegel see https://github.com/easybuilders/easybuild-easyconfigs/pull/19293

**Pull request merged**